content/legal/trademark_guidelines/index.md

```diff
@@ -4,51 +4,46 @@ date: 2018-04-05T16:10:38-04:00
 description: "Supplement to the Eclipse Foundation Guidelines for Eclipse Logos & Trademarks Policy for Jakarta EE Marks"
 hide_sidebar: "false"
 ---
-
-
 <p><em>Supplement to the Eclipse Foundation Guidelines for Eclipse Logos & Trademarks Policy for Jakarta EE Marks</em></p>
 
-<p>Version 1.0 - December 11, 2018</p>
+<p>Version 1.1 - June 13, 2019</p>
 
-<p>This document supplements the Eclipse Foundation <a href="https://www.eclipse.org/legal/logo_guidelines.php">Guidelines for Eclipse Logos & Trademarks
-Policy</a> (this Supplement, together with the Eclipse Foundation Guidelines for Eclipse Logos &
-Trademarks Policy, collectively, the “Policy”) to address the permitted usage of the Jakarta EE
-Marks (as defined below). The Jakarta EE marks include, but are not limited to, the following
-names and/or logos: Jakarta EE, Jakarta EE Working Group, Jakarta EE Member, and Jakarta
-EE Compatible, (collectively the “Jakarta EE Marks”). A full listing of the Jakarta EE Marks may
-be found at: <a href="https://www.eclipse.org/artwork/">https://www.eclipse.org/artwork/</a>. Use of the Jakarta EE Marks must at all times
-conform to the Policy.</p>
+<p>This document supplements the Eclipse Foundation <a href="/legal/logo_guidelines.php" >Guidelines for Eclipse Logos & Trademarks Policy</a> (this Supplement, together with the Eclipse Foundation Guidelines for Eclipse Logos & Trademarks Policy, collectively, the “Policy”) to address the permitted usage of the Jakarta EE Marks (as defined below).  The Jakarta EE marks include, but are not limited to, the following names and/or logos: Jakarta EE, Jakarta EE Working Group, Jakarta EE Member, and Jakarta EE Compatible, (collectively the “Jakarta EE Marks”).  The full guidelines on the usage of the Jakarta EE Marks are described in the <a href="/documents/legal/jakarta-ee-branding-guidelines.pdf" >Jakarta EE Brand Usage Handbook</a>. Use of the Jakarta EE Marks must at all times conform to the Policy. </p>
```

Comment:
I would recommend storing these files in the content/legal/trademark_guidelines/ directory. Please see: https://gohugo.io/content-management/organization/#page-bundles

Reply:
Those files are stored in the same way that other PDFs in the site are stored (insights and membership). Shouldn't we keep those outside of the page structure, as these could be linked outside of these guidelines, and shouldn't be tied to a specific page path?